src/realm/util/interprocess_condvar.cpp

```diff
@@ -190,8 +241,10 @@ void InterprocessCondVar::init_shared_part(SharedPart& shared_part)
 
 void InterprocessCondVar::release_shared_part()
 {
-#ifdef REALM_CONDVAR_EMULATION
+#ifdef REALM_CONDVAR_EMULATION 
+#ifndef _WIN32
     File::try_remove(m_resource_path);
+#endif
 #else
 #endif
```

Comment:
I can write a comment in the empty areas, stating that you must pay attention to it...